packages/grid_client/scripts/applications/nodepilot.ts

```diff
@@ -35,6 +35,7 @@ async function main() {
     sru: instanceCapacity.sru,
     availableFor: grid3.twinId,
     farmId: 1,
+    features: ["ipv4", "wiregaurd", "ip"],
```

Comment:
we need to check what is the current suported name for ipv4

Reply:
ipv4 is deprecated, but since we didn't remove it yet and it's returned in features. I added it

Reply:
so we need to add it in caprover